_pages/en/strategy-build-vs-buy.md

```diff
@@ -118,12 +134,13 @@ Stakeholders are expected to still comply with existing policy instruments inclu
 ### Legal
 
 1. IT Solutions use the ESDC Software Legal Terms Guidance (to be created - see [Coherent set of actions](#coherent-set-of-actions))
-2. IT Solutions and IT Products comply with the ESDC OSS Management Framework
+2. IT Solutions and IT Products comply with the ESDC Open Source Software Management Framework when applicable.
 
 ### Architecture
 
-1. IT Solutions and IT products loose coupling boundaries are at the Business Capability Model level.
-   - Duplication of IT Products or IT Solutions is supported as long as they are for different business capabilities.
+1. IT Solutions and IT products loose coupling boundaries are at intersection of value stream, value stage and business capability (ESDC Business Capability Model).
+   - Duplication of IT Solutions or IT Products is supported as long as they are for different business capabilities.
+   - Boundary of IT Solutions and IT Products are to be application capabilities so that they can enable business capabilities.
 2. IT Products are risk assessed for lock-in situations with the IT Products risk assessment framework (to be created - see [Coherent set of actions](#coherent-set-of-actions))
 
 ## Coherent set of actions
```

Comment:
On item Governance / Update Strategy based on upcoming revision of the Mandatory Procedures on EA and API

Was that a feedback from architecture teams?
It seems redundant to add it as an action for the strategy. If mandatory procedures, it is expected to trigger a review of the strategy.

Suggest removing this action from the list

Reply:
Hmm, well it's an action, not a guiding principle. Action to continuously update the strategy :/ Not sure how we can capture that principle of continuously staying aligned.

Reply:
Removed, kept as mandate in presentation to CIOEC for the team to keep up to date :thinking: